CONTRIBUTING.md

```diff
@@ -64,3 +64,55 @@ Other Tips
     your existing commit message by running:
 
         $ git commit --amend
+
+Checklist for Contributed Code
+------------------------------
+
+A pull request for a new feature will be reviewed to see if it meets the following requirements.  For any pull request, an astropy maintainer can help to make sure that the pull request meets the requirements for inclusion in the package.  
+
+**Scientific Quality**
```

Comment:
I would actually put this a bit further down.  It's not that often we get submissions that do not pass muster on the issue of relevance (when we do it's usually more a question of whether it belongs as an enhancement to a different package like scipy or scikit-image).

Second, many pull requests are for things that are not specifically "science" code, and to which the question of scientific quality does not apply.  Obviously I think this should be in here and I agree with the listed criteria.  It's just not the first thing I would list.